data/templates/responsive/elements/common.xsl

```diff
@@ -164,10 +164,39 @@
         </tr>
     </xsl:template>
 
-    <xsl:template match="tag[@name = 'license' or @name = 'link' or @name = 'see' or @name = 'uses' or @name = 'author']" mode="tabular">
+    <xsl:template match="tag[@name = 'see' or @name = 'uses']" mode="tabular">
+        <xsl:variable name="url">
+            <xsl:choose>
+                <xsl:when test="not(contains(@link, 'http'))">
```

Comment:
@jaapio when an element contains 'http' it will take the 'otherwise' branch on line 173